### PR TITLE
feat(ach withdrawals): added styles and functionality to withdrawal flow

### PR DESCRIPTION
**Pull request by blockdylanb**
## Description (optional)
Adding styles and functionality for withdrawal methods
image
image

## Testing Steps (optional)
- USD enabled account
- Go to USD wallet
- Click Withdraw button in upper right
- click default bank section on enter amount screen

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts [hunk 1]
```diff
@@ -321,6 +321,7 @@ type MessagesType = {
   'copy.available': 'Available'
   'copy.amount_sent': 'Amount Sent'
   'copy.balance': 'Balance'
+  'copy.bank_account': 'Bank Account'
```
Comment (milan-bc) [inline packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts:324]: 👏

File: packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Withdraw/BankPicker/template.success.tsx [hunk 2]
```diff
@@ -81,43 +87,44 @@ const Success: React.FC<Props> = props => {
...
+                style={{ justifyContent: 'flex-start' }}
+              />
+            ) : (
+              <Image
+                name='circle-empty'
+                width='24px'
+                height='24px'
+                style={{ justifyContent: 'flex-start' }}
+              />
+            )}
```
Comment (milan-bc) [inline packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Withdraw/BankPicker/template.success.tsx:111]: we have this code on 2 places, it's not a mandatory but would be cool to make it as one helper function or service and reuse it on both places

**Review by milan-bc — APPROVED** [review]
I tested this one it looks good, great job 👏

File: packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Withdraw/BankPicker/template.success.tsx (hunk 2 above)
Comment (blockdylanb) [reply packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Withdraw/BankPicker/template.success.tsx]: @milan-bc moved it to a service

File: packages/blockchain-wallet-v4-frontend/src/services/ActiveToggleService/index.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        <Image
+          name='circle-empty'
+          width='24px'
+          height='24px'
+          style={{ justifyContent: 'flex-start' }}
+        />
+      )}
+    </>
+  )
+}
```
Comment (milan-bc) [inline packages/blockchain-wallet-v4-frontend/src/services/ActiveToggleService/index.tsx:26]: <3

**Pull request merged**